docs/resources/cce_node_pool.md

```diff
@@ -50,7 +50,8 @@ The following arguments are supported:
 
 * `name` - (Required, String) Node Pool Name.
 
-* `initial_node_count` - (Required, Int) Initial number of expected nodes in the node pool.
+* `initial_node_count` - (Required, Int) Specifies the initial number of expected nodes in the node pool.
+  This parameter is used to manual scaling and conflicts with auto scaling.
```

Comment:
`conflicts` is confusing. Is there any better way to describe the relationship between this parameter and auto scaling

Reply:
seems we can get rid of this line, or maybe update to something like `This parameter can be also used to manually scale the node count afterwards`

Reply:
`This parameter can be also used to manually scale the node count afterwards` is better.